source/api/opensearch.rst

```diff
@@ -11,10 +11,20 @@ The catalog provides an opensearch entry point at http://localhost:8080/geonetwo
 
 .. figure:: img/opensearch.png
 
+Browsers detect the availability of opensearch by checking the index page at the root of the (sub)domain. If you install geonetwork in a subfolder, 
+consider to set up a rewrite rule forwarding the index request to the subfolder.
```

Comment:
I think would be relevant to add an example of such a rule for Apache WebServer or NGIX. I think in most cases the application is installed in a folder of the Java container.